rocksdb_admin/admin_handler.cpp

```diff
@@ -791,15 +949,20 @@ void AdminHandler::async_tm_restoreDBFromS3(
   }
 
   auto ts = common::timeutil::GetCurrentTimestamp();
-  auto local_path = folly::stringPrintf("%ss3_tmp/%s%d/", FLAGS_rocksdb_dir.c_str(), request->db_name.c_str(), ts);
+  auto local_path = FLAGS_enable_checkpoint_backup ? FLAGS_rocksdb_dir + request->db_name :
+                    folly::stringPrintf("%ss3_tmp/%s%d/", FLAGS_rocksdb_dir.c_str(), request->db_name.c_str(), ts);
   boost::system::error_code remove_err;
   boost::system::error_code create_err;
   boost::filesystem::remove_all(local_path, remove_err);
   boost::filesystem::create_directories(local_path, create_err);
-  SCOPE_EXIT { boost::filesystem::remove_all(local_path, remove_err); };
+  SCOPE_EXIT {
+    if (!FLAGS_enable_checkpoint_backup) {
+      boost::filesystem::remove_all(local_path, remove_err);
```

Comment:
does it means after restore, sst files are still stored at /<rocksdb_dir>/s3_tmp/? If true, don't we need to move it out of s3_tmp?

Reply:
no. the files will be downloaded to the original db path